MetasysServicesComExampleApp/Program.cs

```diff
@@ -225,24 +225,30 @@ static void Main(string[] args)
 
             Console.WriteLine("\n List of alarms with details");
 
-            //TODO: write code to get input from console
-            var alarmFilter = new ComAlarmFilter
+            string getAlarms;
+            ComAlarmFilter getAlarmsFilter = new ComAlarmFilter();
+            if (args.Length != 12)
             {
-                StartTime = "2019-12-10T13%3A58%3A20.243Z",
-                EndTime = "2019-12-13T13%3A58%3A20.243Z",
-                PriorityRange = "0%2C255",
-                Type = null,
-                ExcludePending = false,
-                ExcludeAcknowledged = false,
-                ExcludeDiscarded = false,
-                Attribute = null,
-                Category = null,
-                Page = 1,
-                PageSize = 100,
-                Sort = "creationTime"
-            };
-
-            dynamic alarmItems = legacyClient.GetAlarms(alarmFilter);
+                Console.WriteLine("Please enter these parameters separated by space: Start Time, End Time, Priority range, Type, Exclude pending, Exclude acknowledged, Exclude discarded, Attribute, Category, Page, Page Size, Sort" +
+                    "\nRefer to the metasys-server/basic-services-dotnet README if you need help getting started.");
+                getAlarms = Console.ReadLine();
+                args = getAlarms.Split(' ');
+
+                getAlarmsFilter.StartTime = args[0];
+                getAlarmsFilter.EndTime = args[1];
+                getAlarmsFilter.PriorityRange = args[2];
+                getAlarmsFilter.Type = Convert.ToInt32(args[3]);
+                getAlarmsFilter.ExcludePending = Convert.ToBoolean(args[4]);
+                getAlarmsFilter.ExcludeAcknowledged = Convert.ToBoolean(args[5]);
+                getAlarmsFilter.ExcludeDiscarded = Convert.ToBoolean(args[6]);
+                getAlarmsFilter.Attribute = Convert.ToInt32(args[7]);
+                getAlarmsFilter.Category = Convert.ToInt32(args[8]);
+                getAlarmsFilter.Page = Convert.ToInt32(args[9]);
+                getAlarmsFilter.PageSize = Convert.ToInt32(args[10]);
+                getAlarmsFilter.Sort = args[12];
+            }

```

Comment:
Create a private method and assign the value that is read from the console. It is repeating code so the same method will be called to get alarms, get alarm for network devices and objects.